src/Fields/integrals.jl

```diff
@@ -0,0 +1,40 @@
+#####
+##### Column integrals (definite)
+#####
+
+##### Allocating interface:
+column_∫_definite(field::CenterExtrudedFiniteDifferenceField) =
```

Comment:
I'd prefer an ASCII name
```suggestion
column_integrate_definite(field::CenterExtrudedFiniteDifferenceField) =
```

Reply:
How about `column_integral_definite`?

Reply:
I went with `column_integral_definite`, alternatively I'd be fine with `integrate_column_definite`

Reply:
`vertical_integral_definite`?

Reply:
lets keep `column`

Reply:
Discussed more, going with `column_integral_definite`